embeddings/transformation/flair/corpus_transformations.py

```diff
@@ -0,0 +1,94 @@
+import abc
+import pprint
+import tempfile
+from itertools import groupby
+from operator import itemgetter
+from pathlib import Path
+from typing import Dict
+
+import datasets
+from flair.data import Corpus, FlairDataset
+
+from embeddings.transformation.transformation import Transformation
+from experimental.defaults import DATASET_PATH
+
+
+class CorpusTransformation(Transformation[datasets.DatasetDict, Corpus]):
+    HUGGING_FACE_SUBSETS = ["train", "validation", "test"]
+
+    def __init__(
+        self,
+        input_column_name: str,
+        target_column_name: str,
+        datasets_path: Path = DATASET_PATH,
+    ):
+        super().__init__()
+
+        self.datasets_cache_path = datasets_path
+        self.input_column_name = input_column_name
+        self.target_column_name = target_column_name
+
+    def transform(self, data: datasets.DatasetDict) -> Corpus:
```

Comment:
does it always be `hf_datadict`? if yes, then maybe rename it accordingly?

Reply:
But this will change arguments to `transform` defined in abstract class.

Reply:
I think it should remain as it is